assets/messages.yml

```
@@ -590,6 +590,21 @@ messages:
 
         %quick_links%
       fillname: []
+  not-reopen-ar:
+    - project:
+        - MC
+        - MCTEST
+        - MCPE
+        - MCAPI
+        - MCL
+        - MCD
+        - MCE
+        - BDS
+        - REALMS
+        - WEB
+      name: Not reopen AR ticket
+      message: This report will not be reopened automatically. If a moderator sees your comment, they will reopen and update the report. However, if you think your update to this report has been overlooked or you want to make sure that this report is reopened, you can contact the Mojira staff in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/].
```

```suggestion
      message: This report will not be reopened automatically. If you added the required information and a moderator sees your comment, they will reopen and update the report. However, if you think your update to this report has been overlooked or you want to make sure that this report is reopened, you can contact the Mojira staff in [Discord|https://discordapp.com/invite/rpCyfKV] or [Reddit|https://www.reddit.com/r/Mojira/].
```

No moderator will reopen, if it was not actually useful info.